packages/front/src/lib/components/Documents/components/DownloadButton.controller.ts

```diff
@@ -3,10 +3,12 @@ import type { DocumentEntity } from "$lib/entities/DocumentEntity";
 
 export default class DownloadButtonController {
     public downloadBtnLabel: ReadStore<string>;
+    public resetBtnDisabled: ReadStore<boolean>;
 
     constructor(selectDocsStore: ReadStore<DocumentEntity[]>) {
         this.downloadBtnLabel = derived(selectDocsStore, docs =>
             docs.length ? `Télécharger la sélection (${docs.length})` : "Tout télécharger",
         );
+        this.resetBtnDisabled = derived(selectDocsStore, docs => (docs.length > 0 ? false : true));
```

Comment:
```suggestion
        this.resetBtnDisabled = derived(selectDocsStore, docs => docs.length === 0);
```
ou 
```suggestion
        this.resetBtnDisabled = derived(selectDocsStore, docs => !(docs.length > 0));
```

pur être + concis et + lisible